src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/SentenceTypeController.kt

```diff
@@ -69,4 +71,68 @@ class SentenceTypeController(private val sentenceTypesService: SentenceTypeServi
   )
   @ResponseStatus(HttpStatus.OK)
   fun getSentenceTypesByIds(@RequestParam("uuids") uuids: List<UUID>): List<SentenceType> = sentenceTypesService.findByUuids(uuids)
+
```

Comment:
Would these new endpoints better sit in a separate controller?? I'm ythinking this whole piece eels like a 'bolt on' so maybe keep it separate? @carloveo-moj what do you think?

Reply:
Will do!